src/lib/Scenes/Collection/Screens/CollectionArtworkPreview.tsx

```diff
@@ -0,0 +1,40 @@
+import { Separator } from "@artsy/palette"
+import { CollectionArtworkPreview_collection } from "__generated__/CollectionArtworkPreview_collection.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface Props {
+  collection: CollectionArtworkPreview_collection
+}
+
+export class CollectionArtworkPreview extends React.Component<Props> {
+  render() {
+    const { collection } = this.props
+    if (!collection) {
+      return null
+    }
+    const artworks = collection.artworks.edges.map(({ node }) => node)
```

Comment:
Should we use the `get` util against this long chain of properties, to avoid app crashes if for some reason something came back from metaphysics funny?